google/ggrc-core
# GGRC-7669 Fix error when exporting assessments linked to object from trie view

# Dependencies

- [x] This PR is `on hold` until SQL query results from PROD instance are provided.


# Issue description

Export fails when trying to export assessments some object is linked to from trie view. The following error appears: `"Your Export job failed due to a server error. Please restart the export again. Sorry for the inconveniences."`

But when exporting assessments linked to object from export page, everything works fine.

# Steps to test the changes

Changes could be reproduced on PROD instances only due to corrupted data. To reproduce changes locally some manual changing of data in DB is needed:

1. Login in GGRC;
2. Create program, audit;
3. Map control to audit;
4. Create assessment and map control to it;
5. Look in the DB for snapshot of control from step 4;
6. Find revision specified in snapshot and set `resource_slug` to `NULL`;
7. Click on mapped snapshot on UI and select "View original Control" option;
8. Go to "Assessments" tab;
9. Select "Export Assessments" from three dots menu;
10. Go on "Export" page and see that export has failed due to unknown server error.

# Solution description

Solution consists of a migration which updates `resource_slug` for revisions with appropriate value take from revision's content.

# Sanity checklist


# Migration checklist

# PR Review checklist
